Dockerfile.builder

```diff
@@ -10,28 +12,27 @@ LABEL maintainer="Plone Community <dev@plone.org>" \
 RUN <<EOT
     set -e
     apt update
-    apt install -y --no-install-recommends python3 build-essential git ca-certificates
-    npm install --no-audit --no-fund -g yo @plone/generator-volto@alpha
-    mkdir /app
-    chown -R node:node /app
+    apt install -y --no-install-recommends python3 python3-pip build-essential git ca-certificates pipx
+    npm install --no-audit --no-fund -g mrs-developer
     rm -rf /var/lib/apt/lists/*
+    pipx run cookiecutter gh:plone/cookiecutter-volto addon_name=app --no-input
+    chown -R node:node /app
 EOT
 
-WORKDIR /app
-RUN corepack enable
+COPY --chown=node:node volto.config.js /app/.
 
+RUN corepack enable
 USER node
+
+WORKDIR /app
+
 RUN <<EOT
     set -e
-    yo @plone/volto \
-        app \
-        --description "Plone frontend using Volto" \
-        --skip-addons \
-        --skip-install \
-        --skip-workspaces \
-        --volto=${VOLTO_VERSION} \
-        --no-interactive
-    yarn install --network-timeout 1000000
+    sed -i 's/${VOLTO_VERSION}/'"$VOLTO_VERSION"'/g' mrs.developer.json
+    # Removes the addon dependency from package.json
+    python3 -c "import json; data = json.load(open('package.json')); data['dependencies'].pop(list(data['dependencies'].keys())[-1]); json.dump(data, open('package.json', 'w'), indent=2)"
+    rm -rf packages/app
+    make install
```

Comment:
This currently installs all volto packages (including experimental integrations with other frameworks), and dev tools like cypress. It would be nice to cut a lot of that out for the production image.